status/flowcells.py

```diff
@@ -314,6 +322,17 @@ def search_flowcell_names(self, search_string=""):
             except AttributeError:
                 pass
 
+        for row_key in FlowcellSearchHandler.cached_element_fc_list:
+            try:
+                if search_string in row_key.lower():
+                    fc = {
+                        "url": f"/flowcells_element/{row_key}",
+                        "name": row_key,
+                    }
+                    flowcells.append(fc)
+            except AttributeError:
```

Comment:
What attribute error is it that we're catching? The `row_key.lower`? I see that it's the same for all the other instrument types so mostly curious if you know?

Reply:
That makes sense. If you're keen on removing it go for it, but you don't have to for this PR.

Reply:
I'll leave it in place for now.